### ISSUE TITLE
Slicer doesn't slice text

### ISSUE THREAD

**Issue by Geekay1972** [bambulab/BambuStudio#2928]
### Bambu Studio Version

1.7.7.89

### Where is the application from?

Bambu Lab Official website

### OS version

macOS 14.0

### Printer

P1S

### How to reproduce

1. Load the attached STL file in bamboo studio.
2. In the thickness steps every step as a 0.2 mm high text describing the thickness: 0.2, 0.4, 0,6 ....
3. When you slice the model, the text is not in the generated G-code
4. This happens no matter what layer hight you choose, even with a 0.1 layer its lost
[Geeetech_PLA_Silk_gold.stl.zip](https://github.com/bambulab/BambuStudio/files/13418811/Geeetech_PLA_Silk_gold.stl.zip)

### Actual results

<img width="861" alt="Screenshot 2023-11-20 at 20 47 59" src="https://github.com/bambulab/BambuStudio/assets/143263349/80f5f7cc-0c64-49de-8886-6dcb8b554331">
<img width="1440" alt="Screenshot 2023-11-20 at 20 52 24" src="https://github.com/bambulab/BambuStudio/assets/143263349/6c6afd3e-17b4-48e4-a7b4-f2b5ad543615">

### Expected results

numbers should bed printed

### Project file & Debug log uploads

[log.zip](https://github.com/bambulab/BambuStudio/files/13418935/log.zip)
[Geeetech_PLA_Silk_gold.3mf.zip](https://github.com/bambulab/BambuStudio/files/13418938/Geeetech_PLA_Silk_gold.3mf.zip)

### Checklist of files to include

- [X] Log file
- [X] Project file

**Comment by GuzziRaz** [comment]
There's no bug here. The text is simply too small for your settings (or nozzle). Just change to Arachne (or a 0.2 nozzle) and it no longer disappears.

**Comment by Geekay1972** [comment]
Thanks for the answer. Actually I had the very low hight on my list but not the size :-)
Wouldn't it be a great improvement for the slicer to spit some warning in such a case instead of silently dropping the whole text? Could you make a feature request out of this BR?

**Comment by Geekay1972** [comment]
Yes, sure. I mean in this case it's not just a detail but the whole text as an object that is below resolution.
BTW, I also liked your hint to the Arachne mode (could also go into the warning). I would have never found out that by myself ;-)

**Comment by GuzziRaz** [comment]
Arachne was the default for a while but they reverted that due to some problems (not sure what or how often it occurred - I use it a lot and never had a problem). It will likely be the default again once the bugs are ironed out.

**Comment by Geekay1972** [comment]
> Arachne was the default for a while but they reverted that due to some problems (not sure what or how often it occurred - I use it a lot and never had a problem). It will likely be the default again once the bugs are ironed out.

Thank you for this background info. I meanwhile switched over to Arachne and like it much more, seams to produce less gap infill so I was already wondering why its not the default ....

**Comment by walterwongbbl** [comment]
@zhimin-zeng-bambulab
